### ISSUE TITLE
False negative function_body_length with multiline strings

### ISSUE THREAD

**Issue by kasrababaei** [realm/SwiftLint#4585]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

function_body_length is not fired for a function which has a long multiline string but it used to on previous version. Is this intentional?

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
// Outputs nothing but expected: "Function Body Length Violation: Function body should span 75 lines or less excluding comments and whitespace: currently spans 76 lines (function_body_length)"
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 
  0.50.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)? 
  Homebrew
* Paste your configuration file:

```yml
function_body_length:
  warning: 75
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
  No
* Which Xcode version are you using (check `xcodebuild -version`)?
  14.0.1 (14A400)
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
// This does not trigger a violation:
func foo() -> String {
    """
    Sed ut perspiciatis, unde
    omnis iste natus error
    sit voluptatem accusantium
    doloremque laudantium,
    totam rem aperiam eaque ipsa,
    quae ab illo
    inventore veritatis et quasi
    architecto beatae vitae
    dicta sunt, explicabo.
    Nemo enim ipsam
    voluptatem, quia voluptas sit,
    aspernatur aut odit aut
    fugit, sed quia
    consequuntur magni
    dolores
    eos, qui ratione
    voluptatem sequi
    nesciunt, neque
    porro quisquam est,
    qui dolorem ipsum,
    quia dolor sit amet
    consectetur adipisci[ng]
    velit,
    sed quia non
    numquam [do] eius modi tempora
    inci[di]dunt,
    ut labore et dolore
    magnam aliquam
    quaerat voluptatem.
    Ut enim ad minima veniam,
    quis nostrum[d]
    exercitationem ullam corporis
    suscipit laboriosam,
    nisi ut aliquid ex ea
    commodi consequatur?
    [D]Quis autem vel eum i[r]ure
    reprehenderit,
    qui in ea voluptate velit esse,
    quam nihil molestiae
    consequatur, vel illum,
    qui dolorem eum fugiat,
    quo voluptas nulla
    pariatur? [33] At vero
    eos et accusamus et iusto
    odio dignissimos
    ducimus, qui blanditiis
    praesentium voluptatum
    deleniti atque corrupti,
    quos dolores et quas
    molestias excepturi sint,
    obcaecati cupiditate
    non provident, similique
    sunt in culpa,
    qui officia deserunt mollitia
    animi, id est
    laborum et dolorum fuga.
    Et harum quidem
    rerum facilis est et expedita
    distinctio. Nam libero
    tempore, cum soluta nobis
    est eligendi optio,
    cumque nihil impedit,
    quo minus id, quod
    maxime placeat, facere possimus,
    omnis voluptas assumenda
    est, omnis dolor
    repellendus. Temporibus
    autem quibusdam et aut
    officiis debitis aut rerum necessitatibus
    saepe eveniet, ut et voluptates repudiandae sint et
    molestiae non recusandae. Itaque earum
    rerum hic tenetur a sapiente delectus, ut aut reiciendis
    voluptatibus maiores alias consequatur
    aut perferendis doloribus asperiores repellat.
    """
}
```

**Comment by jpsim** [comment]
Thanks for the detailed bug report. Fix in https://github.com/realm/SwiftLint/pull/4587
